test/source/tests/setup.ts

```diff
@@ -744,4 +746,47 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
 
   }
 
+  if (testVariant === 'CONSUMER-MOCK') {
+    ava.default('setup - imported key with multiple alias should show checkbox per alias', testWithBrowser(undefined, async (t, browser) => {
+      expect((await KeyUtil.parse(testConstants.keyMultiAliasedUser)).emails.length).to.equals(2);
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, 'multi.aliased.user@example.com');
+      await SetupPageRecipe.manualEnter(settingsPage, '', {
+        usedPgpBefore: false, submitPubkey: true, naked: true, fillOnly: true, checkEmailAliasIfPresent: true, key: {
+          title: 'multi.aliased.user@example.com',
+          passphrase: 'long enough to suit requirements',
+          armored: testConstants.keyMultiAliasedUser,
+          longid: null // tslint:disable-line:no-null-keyword
+        }
+      }, { isSavePassphraseChecked: false, isSavePassphraseHidden: false });
+      await settingsPage.close();
```

Comment:
Please add tests for `settingsPage.isChecked('@input-email-alias-alias1examplecom')`
and  `settingsPage.isChecked('@input-email-alias-alias2examplecom')`.
I expect alias2 to be checked by default.

Reply:
I have added the same check we've expected like the above. Thank you, sir!